### PR TITLE
Removed maker toolkit setup survey.

### PR DESCRIPTION
**Pull request by jmkulwik**
OLD:
image

NEW:
image

### CODE REVIEW

**Review by ajpal — APPROVED** [review]
👍

**Comment by ryansloan** [comment]
good call adding the support email link!

File: apps/src/lib/kits/maker/ui/SetupChecklist.jsx [hunk 1]
```diff
@@ -222,14 +210,13 @@ export default class SetupChecklist extends Component {
...
     return (
-      <span>
-        <br />
-        Still having trouble? Please{' '}
-        <a href={this.getSurveyURL()}>submit our quick survey</a> about your
-        setup issues.
-      </span>
+      <UnsafeRenderedMarkdown
+        markdown={
+          i18n.stillHavingTrouble() + ' ' + i18n.contactGeneralSupport()
```
Comment (epeach) [inline apps/src/lib/kits/maker/ui/SetupChecklist.jsx]: Will concatenating strings like this work with RTL?

Comment (epeach) [reply apps/src/lib/kits/maker/ui/SetupChecklist.jsx]: What's the reason for using 'UnsafeRenderedMarkdown'?

Comment (jmkulwik) [reply apps/src/lib/kits/maker/ui/SetupChecklist.jsx]: Still looking into this, but in this case, it made more sense to just combine the two messages.

Comment (jmkulwik) [reply apps/src/lib/kits/maker/ui/SetupChecklist.jsx]: Missed the second question earlier. 
contactGeneralSupport() has a link. i.e. `[support@code.org](mailto:support@code.org)`. I checked with Elijah on the best way to include links in i18n strings. Sounds like the best way (if it's a static link) is to use UnsafeRenderedMarkdown. We know this line is safe, but we also want to keep track of where our markdown is and parse it in a standard way.

Comment (epeach) [reply apps/src/lib/kits/maker/ui/SetupChecklist.jsx]: Great! Thanks for following up on that!

**Review by epeach — APPROVED** [review]
LGTM

**Pull request merged**